packages/gafl-webapp-service/src/pages/renewals/identify/identify.njk

```diff
@@ -22,9 +22,11 @@
             'string.pattern.base': { ref: '#postcode', text: 'Your postcode doesn\'t look right. Check and enter again' }
         },
         'referenceNumber' : {
+            'string.empty': { text: 'Enter the last six digits of your licence number', ref: '#ref' },
+            'string.pattern.base': { text: 'The last six digits of your licence number doesn\'t look right. Check and enter again', ref: '#ref' },
```

Comment:
I'm not convinced that we should be validating the permission number in the frontend.  If I change the permission number validator to disallow vowels (due to the risk of generating a number with expletives) then this will prevent existing users from renewing (current licence numbers allow an E).  This also gets messy from a change management perspective if future changes are made.

Reply:
Ok I am inclined to agree but that is a fair bit of rework. I wrote this based on the current status which did not include any rules around vowels. Can we deal with this as a backlog item.